ui.tests/test-module/specs/recaptcha/recaptcha.runtime.spec.js

```diff
@@ -116,4 +119,45 @@ describe("Form Runtime with Recaptcha Input", () => {
             cy.get(`#${id}`).parent().should("not.have.class", bemBlock);
         })
     })
-})
\ No newline at end of file
+
+    function updateEnterpriseConfig(score) {
+        const secretKey = Cypress.env('RECAPTCHA_ENT_API_KEY');
```

Comment:
The name of the environment variable is CYPRESS_RECAPTCHA_ENT_API_KEY, how is this code working

Reply:
> Cypress automatically removes the leading CYPRESS_ or cypress_ from any environment variable name specified in this way.

https://docs.cypress.io/guides/guides/environment-variables#Option-3-CYPRESS_